src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,111 @@
+package duke;
```

Comment:
Could all the packages here be put in a parent directory `duke` to show that they are associated, such that package names will be prefixed with `duke.`, e.g., `duke.duke`? I think it would be clearer.